projects/sdk/src/lib/farm/LibraryPresets.ts

```diff
@@ -139,5 +140,62 @@ export class LibraryPresets {
       this.bean2usdt(fromMode, FarmToMode.INTERNAL) as StepGenerator,
       this.usdt2weth(FarmFromMode.INTERNAL, toMode) as StepGenerator
     ];
+
+    this.wellWethBean = (fromToken: ERC20Token, toToken: ERC20Token, account: string, from?: FarmFromMode, to?: FarmToMode) => {
```

Comment:
A lot of this is hard to follow, I think we could maybe talk through it in voice for me to get a better understanding.